dotCMS/src/main/java/com/dotmarketing/util/DateUtil.java

```diff
@@ -74,6 +75,8 @@
 
 	public static final String UTC = "UTC";
 
+	public static final SimpleDateFormat EXPORTING_DATE_FORMAT = new SimpleDateFormat("yyyy-MM-dd_hh-mm-ss");

```

Comment:
this could be taken from the Config in case some client wants to customize their format

Reply:
Done.